utils/build/docker/dotnet/install_ddtrace.sh

```diff
@@ -32,7 +32,7 @@ else
     tar xzf $(ls datadog-dotnet-apm*.tar.gz) -C /opt/datadog
 fi
 
-apt-get install -y binutils #we need 'strings' command to extract assembly version which is part of binutils package
```

Comment:
[Note to reviewers] This `apt-get install -y binutils` was moved to the dockerfile so it's baked into an image layer (and potentially cached).